test/Microsoft.Sbom.Tool.Tests/IntegrationTests.cs

```diff
@@ -270,12 +355,13 @@ private void GenerateManifestAndValidateSuccess(string testFolderPath, string ma
         Assert.AreEqual(0, exitCode.Value, $"Unexpected failure. stdout = {stdout}");
     }
 
-    private (string arguments, string outputFile) GetValidateManifestArguments(string testFolderPath, string manifestInfoValue = "SPDX:2.2")
+    private (string arguments, string outputFile) GetValidateManifestArguments(string testFolderPath, string manifestInfoValue = "2.2", string complianceStandardValue = "")
     {
         var outputFile = Path.Combine(TestContext.TestRunDirectory, TestContext.TestName, "validation.json");
         var manifestRootFolderName = Path.Combine(testFolderPath, ManifestRootFolderName);
         var arguments = $"validate -m \"{manifestRootFolderName}\" -b \"{testDropDirectory}\" -o \"{outputFile}\"";
-        arguments += $" -mi {manifestInfoValue}";
+        arguments += string.IsNullOrEmpty(manifestInfoValue) ? string.Empty : $" -mi SPDX:{manifestInfoValue}";
```

Comment:
Rather than mixing interpolation and concatenation, can we use all interpolation? Something like this should work:
```
        var manifestInfoArg = string.IsNullOrEmpty(manifestInfoValue) ? string.Empty : $"-mi SPDX:{manifestInfoValue}";
        var complianceArg = string.IsNullOrEmpty(complianceStandardValue) ? string.Empty : $"-cs {complianceStandardValue}";
        var arguments = $"validate -m \"{manifestRootFolderName}\" -b \"{testDropDirectory}\" -o \"{outputFile}\" {manifestInfoArg} {complianceArg}";
```